sdk/js/src/cosmwasm/address.ts

```diff
@@ -35,6 +36,7 @@ export const isNativeDenomNeutron = (denom: string) => denom === "untrn";
 export const isNativeDenomCelestia = (denom: string) => denom === "utia";
 export const isNativeDenomStargaze = (denom: string) => denom === "ustars";
 export const isNativeDenomSeda = (denom: string) => denom === "aseda";
+export const isNativeDenomDymension = (denom: string) => denom === "dym";
```

Comment:
I think this is `udym`, according to the readme https://github.com/dymensionxyz/dymension?tab=readme-ov-file#initializing-dymd

Reply:
good spot, poking around i found [here](https://github.com/dymensionxyz/dymension/blob/9fa05e7f1f0dfc9d6d83ecf3a309c24eda203bee/scripts/src/genesis_config_commands.sh#L82) is `adym` too, in code search on that repo `udym` is always in some `_test` file

Reply:
Confirmed that this should be `adym` with the Dymension team.

Reply:
updated this